src/sign-in-xplat/MainPage.xaml.cs

```diff
@@ -48,36 +52,36 @@ private async void OnSignInClicked(object sender, EventArgs e)
                     .Build();
 
                 // Sign-in user using MSAL and obtain an access token for MS Graph
-                GraphServiceClient graphClient = new GraphServiceClient(s_graphURL,
-                    new DelegateAuthenticationProvider(async (requestMessage) =>
+                async Task<AuthenticationHeaderValue> SignInUserAndGetTokenUsingMSAL()
+                {
+                    IEnumerable<IAccount> accounts = await s_publicClientApp.GetAccountsAsync().ConfigureAwait(false);
+                    IAccount firstAccount = accounts.FirstOrDefault();
+
+                    //Set the scope for API call to user.read
+                    var s_scopes = new string[] { "user.read" };

```

Comment:
Suggestion `graphScope` instead  (Unless the `s_` prefix is still needed for some reason?)

Reply:
thanks for suggestion @ckittel  you're right the convention is now broken

fixed | addressed from [a68c6](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/17/commits/a68c6fbb8fe0c49eb22c34c197414a02f160f99d)